### PR TITLE
refactor: refactor article flag, add default showcase, add readme

### PR DESCRIPTION
**Pull request by colincclark**

### CODE REVIEW

File: packages/article-flag/README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Article Flag
+
+Artivle flags are attached to an article like a lable to indicate the article's
```
Comment (craigbilner) [inline packages/article-flag/README.md]: Article
label

File: packages/article-flag/README.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+status. There are some common statuses that are exported as components from this
+package, such as "new", "exclusive" or "sponsored", but the default component
```
Comment (craigbilner) [inline packages/article-flag/README.md]: a bit odd to name 3/4?

Comment (craigbilner) [inline packages/article-flag/README.md]: perhaps we should also say that the editors/CMS determines the flag for an article and they currently "expire" server side

File: packages/article-flag/README.md (hunk 1 above)
Comment (colincclark) [reply packages/article-flag/README.md]: fixed typos

File: packages/article-flag/README.md (hunk 2 above)
Comment (colincclark) [reply packages/article-flag/README.md]: was my way of not tieing the docs to the code to tightly. Have added the other one

Comment (colincclark) [reply packages/article-flag/README.md]: Not keen on coupling the docs for TC with what happens server side, but have amended

**Pull request merged**